From e34dabfd63cbe365714e54994fb9d2073dba1d98 Mon Sep 17 00:00:00 2001
From: Colin Ian King <colin.king@canonical.com>
Date: Mon, 14 Nov 2016 22:46:43 +0000
Subject: [PATCH 2117/4301] drm/i2c: tda998x: fix spelling mistake

commit 9b2502b6ebc632ff49743b3639ea12d4f08808a5 upstream

Trivial fix to spelling mistake "configutation" to "configuration"
in dev_err message

Signed-off-by: Colin Ian King <colin.king@canonical.com>
Signed-off-by: Russell King <rmk+kernel@armlinux.org.uk>
---
 drivers/gpu/drm/i2c/tda998x_drv.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/gpu/drm/i2c/tda998x_drv.c b/drivers/gpu/drm/i2c/tda998x_drv.c
index da8430b..86f47e1 100644
--- a/drivers/gpu/drm/i2c/tda998x_drv.c
+++ b/drivers/gpu/drm/i2c/tda998x_drv.c
@@ -880,7 +880,7 @@ static int tda998x_audio_hw_params(struct device *dev, void *data,
 	}
 
 	if (audio.config == 0) {
-		dev_err(dev, "%s: No audio configutation found\n", __func__);
+		dev_err(dev, "%s: No audio configuration found\n", __func__);
 		return -EINVAL;
 	}
 
-- 
1.7.5.4

